launchable/utils/session.py

```diff
@@ -0,0 +1,70 @@
+import click
+import os
+import json
+from pathlib import Path
+from typing import Optional
+
+session_file_path = Path('~/.config/launchable/session.json').expanduser()
+
+
+class SessionError(Exception):
+    pass
+
+
+def read_session(build_name: str) -> Optional[str]:
+    if not session_file_path.exists():
+        raise SessionError(
+            "No session file. Please write session {}".format(session_file_path))
+
+    with session_file_path.open('r') as json_file:
+        try:
+            data = json.load(json_file)
+            return data.get("{}:{}".format(build_name, os.getsid(os.getpid())))
+        except json.JSONDecodeError as e:
+            raise SessionError(
+                "Invalid session file. Please remove {}".format(session_file_path))
+
+
+def write_session(build_name: str, session_id: str) -> None:
+    if session_file_path.exists():
+        with session_file_path.open('r') as json_file:
+            try:
+                data = json.load(json_file)
+            except json.JSONDecodeError as e:
+                raise SessionError(
+                    "Invalid session file. Please remove {}".format(session_file_path))
```

Comment:
In this situation I feel a little more certain that the right thing to do is to simply clobber the file and write a new one. Maybe issue a warning at most.